### ISSUE TITLE
power_rev incorrect for odd powers

### ISSUE THREAD

**Issue by dpsanders** [dpsanders/IntervalConstraintProgramming.jl#29]
``` julia
julia> b
[-1.00001e+16, 1]

julia> x
[-1e+08, 1e+08]

julia> IntervalConstraintProgramming.power_rev(b, x, 3)
([-1.00001e+16, 1],[0, 1],[3, 3])
```

**Comment by dpsanders** [comment]
Simpler example:
```
julia> b = -27..27
[-27, 27]

julia> x = -5..5
[-5, 5]

julia> IntervalConstraintProgramming.power_rev(b, x, 3)
([-27, 27],[0, 3.00001],[3, 3])
```
Here, `x` should end up as `[-3,3]`.

**Comment by dpsanders** [comment]
Fixed in #49
